doc/source/docs/user_guide/projections.rst

```diff
@@ -30,7 +30,7 @@ referred to using the authority code ``"EPSG:4326"``.
 - CRS WKT string
 - An authority string (i.e. "epsg:4326")
 - An EPSG integer code (i.e. 4326)
-- A ``pyproj.CRS``
+- A :class:`pyproj.crs <pyproj.crs.CRS>`
```

Comment:
```suggestion
- A :class:`pyproj.CRS <pyproj.crs.CRS>`
```

same below please. the class uses capital letters.

Reply:
Sorry again, I missed this as well. Fixed in the last [commit](https://github.com/geopandas/geopandas/pull/1931/commits/780bb940826f7349ce22839d384097396523fc0b).